builder/builder_test.go

```diff
@@ -80,35 +161,26 @@ func TestBuild(t *testing.T) {
 
 	for description, test := range tests {
 		t.Run(description, func(t *testing.T) {
+			env := setupTestEnvironment(t)
+
+			app := env.app
+			g := env.g
+			pool := env.pool
+			blockStore := env.blockStore
+			txStore := env.txStore
+			ethstatedb := env.ethstatedb
+			eventBus := env.eventBus
```

Comment:
Rather than pulling each field off of the `testEnvironment` struct and storing them as vars after setup, it may be cleaner to just pull them directly off of `testEnvironment` whenever needed. This comment would also apply for the other cases where we call `setupTestEnvironment`